src/Tizen.NUI/src/public/Events/GestureOptions.cs

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using Tizen.NUI.BaseComponents;
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// This is a calss that sets the configuration options of Gestures
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class GestureOptions : Disposable
```

Comment:
It couldn't be a static class without Disposable?
This class and the bound APIs in DALi have no resources.
This comment is just suggestion without deep understanding of the works of GestureOptions and input-options.
Please use better way.
Thank you.